Readme.md

```diff
@@ -0,0 +1,56 @@
+# basictracer-javascript [![NPM version][npm-image]][npm-url] [![build status][travis-image]][travis-url] [![Test coverage][coveralls-image]][coveralls-url]
```

Comment:
I have a bunch of thoughts about the README, but as a thank-you to you for doing all of this work, I can just make README changes in my own followup PR :)